chart/skywalking/values.yaml

```diff
@@ -373,5 +373,34 @@ elasticsearch:
 
   keystore: []
 
+satellite:
+  name: satellite
+  replicas: 1
+  enabled: false
+  image:
+    repository: skywalking.docker.scarf.sh/apache/skywalking-satellite
+    tag: null # Must be set explicitly
+    pullPolicy: IfNotPresent
+  ports:
+    grpc: 11800
+    prometheus: 1234
+  service:
+    type: ClusterIP
+  antiAffinity: "soft"
+  nodeAffinity: {}
+  nodeSelector: {}
+  tolerations: []
+  resources: {}
+    # limits:
+    #   cpu: 4
+    #   memory: 8Gi
+    # requests:
+    #   cpu: 4
+    #   memory: 4Gi
+  podAnnotations:
+    # example: oap-foo
+  env:
+    # more env, please refer to https://hub.docker.com/r/apache/skywalking-satellite
```

Comment:
There is nothing env can be referred in this link https://hub.docker.com/r/apache/skywalking-satellite

Reply:
fix to using official documentation for satellite.